client/modules/users/components/SignUp.js

```diff
@@ -138,10 +145,14 @@ class SignUp extends React.Component {
           {password.touched && passwordConfirm.touched && password.value !== passwordConfirm.value &&
             <div className="alert alert-danger">Passwords do not match</div>
           }
+          {password.touched && (password.value.length < 7 && passwordConfirm.value.length < 7) &&
```

Comment:
less important than the above, but I think it would be better if only one error was shown at a time - if the password requirements aren't met show the password must be ... error, otherwise show the passwords don't match error if appropriate

Reply:
done, thanks for the review